blocks/api/registration.js

```diff
@@ -288,3 +288,25 @@ export function hasBlockSupport( nameOrType, feature, defaultSupports ) {
 export function isSharedBlock( blockOrType ) {
 	return blockOrType.name === 'core/block';
 }
+/**
+ * Returns an array with the child blocks of a given block.
+ *
+ * @param {string} blockName Block type name.
+ *
+ * @return {Array} Array of child block names.
+ */
```

Comment:
Not sure a JSDoc parser would detect this as being "attached" to the `getChildBlockNames` function with the newline between.

Reply:
The line breaks were corrected.